build-tasks/common/meta.js

```diff
@@ -80,23 +80,7 @@ function getContribRuleNames() {
         .sort();
 }
 
-function getAllFormatterNames() {
```

Comment:
Not related changes?

Reply:
Looks like should be part of #638 and this is wrong branch.

Reply:
Agh, thanks...